operations/billingautomations.md

```diff
@@ -0,0 +1,591 @@
+| `AssignmentsToAdd` | array of [Billing automation assignment add parameters](billingautomations.md#billing-automation-assignment-add-parameters) | optional, max 20 items | List of parameters adding new assignments (or null should it not be updated). |
+| `AssignmentsToUpdate` | array of [Billing automation assignment update parameters](billingautomations.md#billing-automation-assignment-update-parameters) | optional, max 20 items | List of parameters updating existing assignments (or null should it not be updated). |
+| `AssignmentIdsToRemove` | array of string | optional, max 100 items | List of unique identifiers of assignments, which should be removed (or null should it not be updated). |
+
+### Response
+
+```javascript
+{
+  "BillingAutomations": [
+    {
+      "Id": "ecd64eec-4423-4c65-b844-814b9199856d",
+      "EnterpriseId": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+      "Name": "Continuous billing automation for company XYZ",
+      "Description": "Billing Automation for company XYZ with prepayment all, company as owner.",
+      "CompaniesWithRelations": [
+        {
+          "CompanyId": "c6f5c82d-621a-4c8a-903b-1b0a9a23b71f",
+          "CompanyRelations": {
+            "PartnerCompany": true,
+            "TravelAgency": false
+          }
+        }
+      ],
+      "Prepayment": "All",
+      "AssignmentTargetType": "CompanyAsOwner",
+      "TriggerType": "Continuous",
+      "BillAggregationType": "AggregateAll",
+      "CreatedUtc": "2017-01-31T10:58:06Z",
+      "ProcessingStartOffset": null,
+      "OrderItemConsumptionPeriod": null,
+      "Assignments": null
+    }
+  ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `EnterpriseId` | string | optional | Unique identifier of the enterprise. Required when using [Portfolio Access Tokens](../concepts/multi-property.md), ignored otherwise. |
+| `BillingAutomationId` | string | required | Unique identifier of the billing automation. |
+| `AssignmentsToAdd` | array of [Billing automation assignment add parameters](billingautomations.md#billing-automation-assignment-add-parameters) | optional, max 20 items | List of parameters adding new assignments (or null should it not be updated). |
+| `AssignmentsToUpdate` | array of [Billing automation assignment update parameters](billingautomations.md#billing-automation-assignment-update-parameters) | optional, max 20 items | List of parameters updating existing assignments (or null should it not be updated). |
+| `AssignmentIdsToRemove` | array of string | optional, max 100 items | List of unique identifiers of assignments, which should be removed (or null should it not be updated). |
```

Comment:
Response type is wrong. This needs to be fixed in the API contracts. There is also a wrong method name (UpdateRoutingRulesAssignmentsAsync) there.